app/src/main/res/layout/activity_find_me.xml

```diff
@@ -13,20 +13,23 @@
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:layout_weight="15"
-        android:background="#00FF0A"
-        android:contentDescription="Map logo"
-        map:srcCompat="@android:drawable/ic_dialog_map" />
+        android:background="@color/primary"
+        android:contentDescription="@string/map_logo"
+        map:srcCompat="@drawable/ic_map"
+        map:tint="@color/white"
+        tools:ignore="VectorDrawableCompat" />
```

Comment:
I think the ignore in line 20 is not needed because of srcCompat

Line 30: android:inputType is for EditText only. 

Also the "My Location" is sticking out the right of the screen.